stack-graphs/tests/it/test_graphs/simple.rs

```diff
@@ -77,6 +78,7 @@ pub fn new() -> StackGraph {
         syntax_type: str_var.into(),
         containing_line: str_line0.into(),
         definiens_span: Span::default(),
+        fully_qualified_name: None.into(),
```

Comment:
super nit: You're using `None.into` here but `ControlledOption::default` below. Can you make them consistent?

Reply:
👍 oops, thanks! I've pushed an update that fixes this and updates the `serde` contexts.